Docker/pytools/Dockerfile

```diff
@@ -15,9 +15,7 @@ USER admin
 
 ENV HOME=/home/admin \
     PYENV_ROOT=$HOME/.pyenv \
-    PATH=$HOME/.local/bin:/home/admin/.pyenv/shims:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
-    PATH=$PATH:/home/admin/.pyenv/bin:$PYENV_ROOT/plugins/pyenv-virtualenv/shims \
-    PATH=$PATH:$HOME/.pyenv/shims:$HOME/bin:/$HOME/.poetry/bin \
+    PATH=/home/admin/.pyenv/shims:/home/admin/.pyenv/bin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/admin/.poetry/bin:. \
```

Comment:
The PATH variable don't have the next directory $HOME/.local/bin

Reply:
Ok, fixed